pulsar/consumer_test.go

```diff
@@ -41,7 +41,7 @@ import (
 
 var (
 	adminURL  = "http://localhost:8080"
-	lookupURL = "pulsar://localhost:6650"
+	lookupURL = "pulsar://10.105.7.225:6650"
```

Comment:
Is this right to change to a fixed private IP? Would that work in CI and other localhost environment?

Reply:
I have find the real reason. https://github.com/apache/pulsar-client-go/pull/875

It's a consumer bug caused by availablePermitsCh.